core/src/allocation_init.jl

```diff
@@ -684,6 +779,26 @@ function add_constraints_fractional_flow!(
     return nothing
 end
 
+"""
+Add the basin flow constraints to the allocation problem.
+The constraint indices are the basin node IDs.
+NOTE: negative basin flow means flow out of the basin.
+
+Constraint:
+flow basin >= - basin capacity
```

Comment:
Can you explain this constraint a bit more? How is basin flow and capacity defined? What does a negative capacity mean? And how does it relate to the 0.0 in the code below?

Reply:
I updated the description. This was from when I still used a single variable for flow in and out of the basin, now I have separate variables for that. The $0.0$ below does not have a specific meaning, it's a dummy value to initialize the constraint.